src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -184,6 +185,10 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.ConfigureOptionsWithChangeTokenSource<AppServiceOptions, AppServiceOptionsSetup, SpecializationChangeTokenSource<AppServiceOptions>>();
             services.ConfigureOptionsWithChangeTokenSource<HttpBodyControlOptions, HttpBodyControlOptionsSetup, SpecializationChangeTokenSource<HttpBodyControlOptions>>();
 
+            //ServelessSecurityDefender
```

Comment:
Do you need this comment?

Reply:
Yeah,  followed other Services registering in this class.